src/CEOFESABundle/Controller/StructureController.php

```diff
@@ -239,4 +241,28 @@ private function createDeleteForm($id)
             ->getForm()
         ;
     }
+
+    /**
+     * @param $id
+     *
+     * @Route("/editpdf/{id}", name="structure_pdf")
+     * @return Response
+     */
+    public function createPdf($id)
+    {
+        $em = $this->getDoctrine()->getManager();
+
+        $entity = $em->getRepository('CEOFESABundle:Structure')->find($id);
+
+        $html = $this->renderView('::Templates\convention_partenariat.html.twig', array(
+            'info' => $entity
+        ));
```

Comment:
CS
You have only one item in your array AND it all fits on one line so put it inline :)
And in case of multiple items, you need to put a `,`, even if it's the array's last item :)